irakozetony/my-brand
# #Oe5xeXrc implementing all pages and making responsive

### what does this PR do?
This PR implements the portfolio page, contact page, blogs page, single article page, login page, dashboard, and the create new article page

### description of tasks to be completed
- create the contact page
- create the portfolio page
- create the blogs page
- create the single article page
- create the login page
- create the dashboard page
- create the new article page

### how should this be manually tested
- clone the [repo](https://github.com/irakozetony/my-brand/)
- checkout branch 'feat-portfolio-page-Oe5xeXrc'
- open the project with live-server and check the pages

### What are the relevant pivotal tracker stories
[trello](https://trello.com/c/Oe5xeXrc/14-implement-other-pages-and-make-responsive) link
